src/components/ScaleControls.vue

```diff
@@ -1,18 +1,65 @@
 <script setup lang="ts">
 import { useScaleStore } from '@/stores/scale'
+import { useStateStore } from '@/stores/state'
 import { debounce, midiNoteNumberToName } from '@/utils'
 import { ref } from 'vue'
 import ScaleRule from './ScaleRule.vue'
 import palette from '@/character-palette.json'
 
+const state = useStateStore()
 const scale = useScaleStore()
-
 const updateScale = debounce(scale.computeScale)
 
 const sourceEditor = ref<HTMLTextAreaElement | null>(null)
-
 const paletteInfo = ref('')
 
+function clearUserNotation() {
+  scale.userNotation = ''
+  updateScale()
+}
+
+function updateScoreChordOctave(step: number) {
+  console.log('--------------------')
```

Comment:
Is this PR a draft or intended for production? Please no spam to users' consoles in production or even staging.